### ISSUE TITLE
Fatal error: CERT_UNTRUSTED

### ISSUE THREAD

**Issue by RainbowDashDC** [atom/grunt-download-atom-shell#17]
On ANTERGOS, however applies to Arch Linux in general currently.

Uname: 'Linux ANTERGOS 3.17.6-1-ARCH #1 SMP PREEMPT Sun Dec 7 23:43:32 UTC 2014 x86_64 GNU/Linux'

grunt -v in build/
```
Running "download-atom-shell" task
Verifying properties download-atom-shell.version, download-atom-shell.outputDir exist in config...OK
Fatal error: CERT_UNTRUSTED
```

**Comment by yevmel** [comment]
works with `0.10.35` again... just update

**Comment by kevinsawicki** [comment]
Yeah, I'm pretty sure this was a node issue that will be fixed by upgrading to the latest 0.10.x version.
